src/components/Bookings.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults";
+import CustomerProfile from "./CustomerProfile.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [customerProfile, setCustomerProfile] = useState([]);
+
+  useEffect(() => {
+    const getBookings = async () => {
+      const res = await fetch("https://cyf-react.glitch.me");
+
+      if (res.status === 200) {
+        const data = await res.json();
+        setBookings(data);
+
+        setLoading(false);
+      } else {
+        setLoading(false);
+        setError(true);
+      }
+    };
+
+    getBookings();
+  }, []);
+
+  const search = searchVal => {
+    searchVal = searchVal.toLowerCase();
+    setBookings(
+      bookings.filter(
+        booking =>
+          booking.firstName.toLowerCase() === searchVal ||
+          booking.surname.toLowerCase() === searchVal
+      )
+    );
+  };
+
+  const showProfile = async id => {
+    const res = await fetch(`https://cyf-react.glitch.me/customers/${id}`);
+
+    if (res.status === 200) {
+      const data = await res.json();
+      setCustomerProfile(data);
+    }
+  };
+
+  return (
+    <div className="App-content">
+      <div className="container-fluid">
+        <Search search={search} />
+        <div className="cont-search-results">
+          {loading ? <h1>Loading...</h1> : <></>}
```

Comment:
```suggestion
          {loading ? <h1>Loading...</h1> : null}
```
Is rendering `null` preferable here? Not completely sure myself - just makes more sense to me to say render nothing in the case where the left hand condition evaluates to `false`. I might be wrong though.

Reply:
how about this instead?
```suggestion
          {loading && <h1>Loading...</h1>}
```